examples/Politicsnews.py

```diff
@@ -1,8 +1,8 @@
-from teheran_news import Teheran_news
+from teheran_news.teheran_news import Teheran_news
 from itertools import islice
 
 
-# Get 32 news of politics
+# Get 32 news of politics, you can choose another topic (Economy, Society, Sports, International or Culture)
 
 articles = Teheran_news('Politics')
 for news in islice(articles.info, 32):
```

Comment:
PEP8 recommend leaving a blank line at the end of every .py file. 

Also read blank lines section. 
E.g Use 2 blank lines between import section, classes, functions, etc
https://www.python.org/dev/peps/pep-0008/#blank-lines